data/changelog.txt

```diff
@@ -1,30 +1,115 @@
 # 0.H
 
 ## Features:
+Prevent occlusions via (auto)-semi-transparent sprites (similar to retraction)
+flip overmap tiles to default true
+Parametric basecamp expansion definitions
```

Comment:
I don't know enough about the feature, but this summary line doesn't tell me much

Reply:
```suggestion
Add parametric mapgen to consolidate similar basecamp definitions
```